data/deep missions.txt

```diff
@@ -495,6 +509,7 @@ mission "Deep: Remnant 2A"
 	
 	on fail
 		event "reset arculus"
+		# add a backup mission? if the Remnant is hostile, scanning could be difficult.
```

Comment:
It's intended to be. If the player can't handle scanning two ships which are hostile to them then they won't be able to plunder any Remant tech later down the road. 
Not sure what the backup mission would entail.